### DISCUSSION TITLE
ABSL_ATTRIBUTE_NO_UNIQUE_ADDRESS doesn't really work for MSVC

### DISCUSSION THREAD

**Discussion by MBkkt** [abseil/abseil-cpp#1418] [General]
https://godbolt.org/z/bf8caE5Tr

It mostly fault of msvc, as I know they don't fix it because some abi issue, but I'm not sure.

Anyway will be nice to have workaround for this.

WDYT?

**Comment by derekmauro** [comment]
We are in a position to workaround this since we don't claim ABI stability in mixed-mode builds. We can use the workaround recommended by Microsoft in https://devblogs.microsoft.com/cppblog/msvc-cpp20-and-the-std-cpp20-switch/#c20-no_unique_address.

**Reply by MBkkt** [reply]
Ok, thanks you

**Reply by derekmauro** [reply]
Addressed by b6de7b80325514018d38de2c4dee1254258c4b31.
